test/aws-lambda/features/unhandled.feature

```diff
@@ -91,13 +117,12 @@ Scenario Outline: unhandled exceptions are reported when using serverless-expres
 Scenario: unhandled asynchronous exceptions are reported when using serverless-express
     Given I setup the environment
     When I invoke the "ExpressFunction" lambda in "features/fixtures/serverless-express-app" with the "events/unhandled-async.json" event
-    Then the lambda response "errorMessage" equals "busted"
```

Comment:
The domain error handler was (incorrectly) preventing the application from crashing and returning a response. Now, the exception goes to the top-level uncaught exception handler before the process exists as it should